.travis.yml

```diff
@@ -13,4 +13,5 @@ matrix:
     - rvm: jruby
   fast_finish: true
 
+before_install: "gem update bundler"
```

Comment:
Ah, I see where this is coming from. Pardon my french, but you fucked up the rebase.
This change and a **lot** of other changes in this PR are coming from the one [with the ConsoleReport](https://github.com/whitesmith/rubycritic/pull/91) we recently merged.
I'm afraid you'll have to remove all changes that slipped into your rebase before we can merge this.
The joys of git ;)